tests/cross-compile/glsl-generic-in.slang.glsl

```diff
@@ -0,0 +1,70 @@
+//TEST_IGNORE_FILE:
+#version 450
+layout(row_major) uniform;
+layout(row_major) buffer;
+
+#line 12 0
+struct F_0
+{
+    vec4 v0_0;
+    vec2 v1_0;
+};
+
+
+#line 16
+float F_get_0(F_0 this_0)
+{
+
+#line 16
+    return this_0.v0_0.x + this_0.v1_0.x;
+}
+
+
+float E_get_0()
+{
+
+#line 20
+    return 1.00000000000000000000;
+}
+
+
+#line 6
+layout(location = 0)
+in vec3 _S1;
+
+layout(location = 1)
+in vec4 _S2;
+
+layout(location = 2)
+in vec2 _S3;
+
+struct GIn_0
+{
+    vec3 p0_0;
+    F_0 field_0;
+};
+
+struct VOut_0
+{
+    vec4 projPos_0;
+};
+
+
+
+void main()
+{
+    GIn_0 _S4 = GIn_0(_S1, F_0(_S2, _S3));
+
+#line 30
```

Comment:
I maybe have missed this in the other PRs, but we should probably not leave `#line` directives in these baselines, since we aren't trying to validate anything around line information in the generated SPIR-V.